docs/contributing/code-style.md

```diff
@@ -152,41 +152,40 @@ all code via our [linter tool](../testing/linters.md) with
 `tools/lint --only=prettier --fix`. You can also [integrate it with your
 editor](https://prettier.io/docs/en/editors.html).
 
-Combine adjacent on-ready functions, if they are logically related.
-
-The best way to build complicated DOM elements is a Handlebars template
+- Combine adjacent on-ready functions, if they are logically related.
```

Comment:
This advice is probably stale -- we no longer use on-ready functions as a habit at all, in favor of the `ui_init.js` module calling explicit initialize methods for each module.